src/controllers/base.js

```diff
@@ -17,6 +17,14 @@ export const horizontalDefaults = {
   }
 };
 
+export function toFixed(value) {
+  const decimals = this._chart.config.options.tooltipDecimals; // inject number of decimals from config
+  if (!decimals || typeof decimals !== 'number' || decimals < 0) {
```

Comment:
This doesn't work as expected. `!decimals` will also be true for `tooltipDecimals: 0` and return the full `value`. As developer I would expect, that the function returns an integer. 

The fix would be explicitly checking for `null` and `undefined`:

```js
if (decimals === null || decimals === undefined || typeof decimals !== 'number' || decimals < 0) {
```

Reply:
due to coercion `decimals == null` will check for both `null` and `undefined`